internal/commands/http-routes/command.go

```diff
@@ -0,0 +1,40 @@
+package http_routes
+
+import (
+	"github.com/mitchellh/cli"
+)
+
+func New() *Command {
+	return &Command{}
+}
+
+type Command struct{}
+
+func (c *Command) Run(args []string) int {
+	return cli.RunResultHelp
+}
+
+func (c *Command) Synopsis() string {
+	return synopsis
+}
+
+func (c *Command) Help() string {
+	return help
+}
+
+const synopsis = "Manage Consul API Gateway HTTPRoutes"
+const help = `
+Usage: consul-api-gateway http-routes <subcommand> [options] [args]
+  This command has subcommands for interacting with Consul API Gateway
+  TCPRoute configuration objects. Here are some simple examples, and more detailed examples are
+  available in the subcommands or the documentation.
```

Comment:
Same comment as my last, in reference to adding a link to navigate to the proper documentation.